.github/workflows/lint.yml

```diff
@@ -53,10 +43,15 @@ jobs:
           sudo apt-get install -y -qq packer
       - name: Packer validate
         run: |
-          packer validate packer/ubuntu-build.json
-          packer validate packer/mint-build.json
           # Allow beta/test builds to fail validation
-          packer validate -var-file=packer/beta-vars.json packer/mint-build.json || true
+          packer validate -var-file=packer/mint-beta.pkrvars.hcl . || true
```

Comment:
I just noticed this step fails because we're already in the packer directory, so the file reference is invalid.

Reply:
Ahh damn `|| true`!